### DISCUSSION TITLE
377628-Microsoft Visual Studio Code Jupyter Extension Elevation of Privilege Vulnerability for October 2022

### DISCUSSION THREAD

**Discussion by DryadYao000** [microsoft/vscode-jupyter#12028] [Questions and Answers]
There is a vulnerability when using the Visual Studio Code Jupyter Extension. We tried to remediate by deleting the extension, or install the latest version of Jupyer extension, however issue still persist.
![image](https://user-images.githubusercontent.com/116635952/201863461-13fc098e-e8de-4563-bffb-af2fcc444d56.png)

**Comment by DonJayamanne** [comment]
Thsi should be fixed in the latest release.

**Reply by DryadYao000** [reply]
Hello Don, thank you very much for your response. I have tried to update the Jupyter or remove this extension, but this issue still appears.

**Comment by DonJayamanne** [comment]
> have tried to update the Jupyter 

please could you provide the following
* version installed
* output from the jupyter output panel 
* exact steps to reproduce the error


> remove this extension, but

I'm sorry i don't follow you here.
what is the error you are getting after the extension is installed, ie how are you running into a problem
